springdoc-openapi-common/src/main/java/org/springdoc/core/SpringDocConfiguration.java

```diff
@@ -387,13 +387,13 @@ OperationCustomizer actuatorCustomizer() {
 		}
 
 		/**
-		 * Actuator customizer OpenAPI customiser.
+		 * Actuator customizer OpenAPI customizer.
 		 *
-		 * @return the OpenAPI customiser
+		 * @return the OpenAPI customizer
 		 */
 		@Bean
 		@Lazy(false)
-		OpenApiCustomiser actuatorOpenApiCustomiser(WebEndpointProperties webEndpointProperties) {
+		OpenApiCustomizer actuatorOpenApiCustomizer(WebEndpointProperties webEndpointProperties) {
```

Comment:
❗may cause compatibility issues because of method renaming, e.g. if class is inherited (will raise compilation issue if `@Override` annotation is used)